terraform/runner/variables.tf

```diff
@@ -4,72 +4,79 @@ variable "cf_password" {
 variable "cf_user" {}
 
 variable "cf_org_name" {
-  type = string
-  default = ""
+  type        = string
+  default     = ""
   description = "Cloud Foundry Organization"
 }
 
 variable "cf_space_name" {
-  type = string
-  default = ""
+  type        = string
+  default     = ""
   description = "Cloud Foundry Space"
 }
 
 variable "ci_server_token" {
-  type = string
-  default = ""
-  sensitive = true
+  type        = string
+  default     = ""
+  sensitive   = true
   description = "Gitlab CI Server Token"
 }
 
 variable "ci_server_url" {
-  type = string
-  default = "https://gitlab.com/"
+  type        = string
+  default     = "https://gitlab.com/"
   description = "Gitlab URL"
 }
 
 variable "default_job_image" {
-  type = string
-  default = "ubuntu:jammy"
+  type        = string
+  default     = "ubuntu:jammy"
   description = "Default Job Image"
 }
 
 # Two executors are supported:
 #  custom - Runs jobs in new application instances, deleted after the run.
 #  shell - Runs jobs directly on the Runner manager.
 variable "runner_executor" {
-  type = string
-  default = "custom"
+  type        = string
+  default     = "custom"
   description = "Runner Executer"
 }
 
 variable "runner_name" {
-  type = string
-  default = "gitlab-runner"
+  type        = string
+  default     = "gitlab-runner"
   description = "Cloud Foundry Organization"
 }
 
 variable "runner_memory" {
-  type = number
-  default = 512
+  type        = number
+  default     = 512
   description = "Manager Runner Memory"
 }
 
 variable "worker_memory" {
-  type = string
-  default = ""
```

Comment:
Question (non-blocking): should this be 512M to match the `var.tfvars`, or, why is it different?

Reply:
Ah, yes! Fixed.